lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+# deals with menus and selection
+
+class Menu
+
+	attr_reader :dishes, :prices
+
+	def initialize
+		@dishes = Hash.new
+		@prices = prices
+	end
+
+	def list_dishes
+		@dishes
+	end
+
+	def prices
```

Comment:
Style/TrivialAccessors: Use `attr_reader` to define trivial reader methods. (https://github.com/bbatsov/ruby-style-guide#attr_family)